.github/workflows/release.yml

```diff
@@ -13,6 +13,12 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}
   cancel-in-progress: true
 
+permissions:
+  contents: write # to be able to publish a GitHub release
+  id-token: write # to enable use of OIDC for npm provenance
+  issues: write # to be able to comment on released issues
+  pull-requests: write # to be able to comment on released pull requests
+
```

Comment:
I'm unsure we need these - we should get write permission by default based on the repo settings - which looks accurate on the latest prerelease workflow run:

<img width="300" alt="Screenshot 2025-08-06 at 10 18 33 AM" src="https://github.com/user-attachments/assets/74c53c97-33a3-43e6-a566-fb4f7cac50ee" />

They do specifically mention `id-token` and I don't see an equivalent in the list there so it's possible we do need that one.  Can we move it down to the specific `release` job so it's not applying to all jobs in the file?

We also manually publish our Github releases so I don't think `contents` applies here?